content/hi/docs/contribute/_index.md

```diff
@@ -0,0 +1,178 @@
+---
+content_type: concept
+title: Contribute to K8s docs
+linktitle: Contribute
+main_menu: true
+no_list: true
+weight: 80
+card:
+  name: योगदान
+  weight: 10
+  title: कुबेरनेटेस मे योगदान देना शुरू करे
+
+---
+
+<!-- अवलोकन  -->
+
+*कुबेरनेटस सभी योगदानकर्ताओ से सुधार का स्वागत करता है चाहे वो नए हो या अनुभवी   *
+
+{{< ध्यान दे  >}}
+कुबेरनेटस मे योगदान करने के बारे मे ज्यादा जानकारी के लिए 
+[योगदानकर्ता प्रलेखन ](https://www.kubernetes.dev/docs/) देखे । .
+{{< /ध्यान दे  >}}
+
+ये वेबसाईट  [कुबेरनेटस सिग डॉक्स ](/docs/contribute/#get-involved-with-sig-docs) द्वारा  देख रेख की जाती है ।
```

Comment:
This sentence isn't correctly formed. The correct would be :
`यह वेबसाईट कि देख रेख [कुबेरनेट्स सिग डॉक्स ](/docs/contribute/#get-involved-with-sig-docs) द्वारा की जाती है ।`

Reply:
thanks, @jayesh-srivastava . The suggestion is now being implemented.

Reply:
I think कुबेरनेटस is the right word to go with. As suggested by @divya-mohan0209